src/features/MoviePage/MoviePage.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import Rating from '../../components/Rating/Rating';
+import CustomButton from '../../components/CustomButton/CustomButton';
+import Spinner from '../../components/Spinner/Spinner';
+import { fetchDeleteMovie, fetchMovieById, fetchActors } from '../../redux/movies/movies.async.actions';
+import { actorsDataSelector, activeMovieDataSelector } from '../../redux/movies/movies.selectors';
+import { propTypesShapes } from '../../constants';
+import styles from './MoviePage.module.scss';
+
+
+class MoviePage extends Component {
+  componentDidMount = () => {
+    const { fetchMovieById, fetchActors, match } = this.props;
+
+    fetchMovieById(Number(match.params.id));
+    fetchActors()
+  };
+
+  getActorsLinks = (actorIds, actorsData) => actorIds.map((actorId, index) => {
+    const actorData = actorsData.find((actor) => actor.id === actorId);
+    if (index === actorIds.length - 1) {
+      return (
+        <React.Fragment key={index}>
+          <Link className={styles.link} to={`/actor/${actorData.id}`}>{actorData.name}</Link>
+        </React.Fragment>
+      )
+    } else {
+      return (
+        <React.Fragment key={index}>
+          <Link className={styles.link} to={`/actor/${actorData.id}`}>{actorData.name}</Link><span>, </span>
+        </React.Fragment>
+      )
+    }
+  });
+
+  deleteMovieHandler = (id) => {
+    const { history, fetchDeleteMovie } = this.props;
```

Comment:
same about redundant descruct, sometimes it's useful(deep nesting, etc.), but not here.